### PR TITLE
Remove LongPress report and simply have Press and Release

### PR DESCRIPTION
**Pull request by bschwind**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -150,13 +151,14 @@ impl Report {
                 },
                 2 + message.len(),
             ))),
-            [header, ..] if b"VPLXED".contains(&header) => Ok(None),
+            [header, ..] if b"VED".contains(&header) => Ok(None),
```
Comment (skywhale) [inline src/lib.rs:154]: why this change?

Comment (bschwind) [reply src/lib.rs:154]: Reports which are only 1 byte long don't need to be included here as they'll be caught in the patterns above.

Comment (skywhale) [reply src/lib.rs:154]: ah I see :)
